### PR TITLE
Test against Ruby 2.2 and 2.3, Rails 4.2 and 5

### PR DESCRIPTION
**Pull request by sferik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ Add this line to your application's Gemfile:
 gem "interactor-rails", "~> 2.0"
 ```
 
-Interactor Rails is compatible with Ruby 1.9, 2.0 or 2.1 on Rails 3 or 4.
+Interactor Rails is compatible with Ruby 1.9.3, 2.0 2.1, 2.2, or 2.3 on Rails
```
Comment (laserlemon) [inline README.md]: Missing a comma.

File: .travis.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@ before_install:
 branches:
   only:
     - master
+cache: bundler
```
Comment (laserlemon) [inline .travis.yml]: :+1:

File: .travis.yml [hunk 3]
```diff
@@ -15,13 +16,22 @@ gemfile:
...
+      gemfile: gemfiles/rails50.gemfile
 rvm:
   - 1.9.3
   - "2.0"
   - "2.1"
+  - 2.2.2
+  - 2.3.0
   - ruby-head
 script: bundle exec rspec
+sudo: false
```
Comment (laserlemon) [inline .travis.yml]: :+1:

File: .travis.yml [hunk 4]
```diff
@@ -15,13 +16,22 @@ gemfile:
...
+  include:
+    - rvm: 2.2.2
+      gemfile: gemfiles/rails50.gemfile
+    - rvm: 2.3.0
+      gemfile: gemfiles/rails50.gemfile
 rvm:
   - 1.9.3
   - "2.0"
   - "2.1"
+  - 2.2.2
```
Comment (laserlemon) [inline .travis.yml]: Any reason not to specify `2.2.4`?

Comment (gaffneyc) [inline .travis.yml]: Does Travis support "2.2" like it does for 2.0 and 2.1?

Comment (laserlemon) [inline .travis.yml]: Last time I checked, no. It's a known issue though so that might have changed.

File: README.md (hunk 1 above)
Comment (sferik) [inline README.md]: Fixed.

File: .travis.yml (hunk 4 above)
Comment (sferik) [inline .travis.yml]: Rails 5 requires Ruby >= 2.2.2. When you declare Ruby 2.2 in `travis.yml`, Travis currently installs Ruby 2.2.0. I could have gone with 2.2.4 but I don’t actually think it matters much. If the tests pass on 2.2.2 but fail on 2.2.4, that’s a problem with Ruby, not this library.

**Comment by sferik** [comment]
@laserlemon @gaffneyc Any other feedback here? I fixed the one typo in the `README`.

**Comment by laserlemon** [comment]
No, looks great, thank you!

**Comment by laserlemon** [comment]
Version 2.0.2 released. Thanks again!

**Pull request merged**